apps/ejabberd/src/ejabberd_rdbms.erl

```diff
@@ -30,7 +30,7 @@
 -export([start/0, start_pool/1, stop_pool/1, pools/0]).
 -include("ejabberd.hrl").
 
--spec start() -> 'ok' | {'error', 'lager_not_running'}.
+-spec start() -> 'ok' | {'error', Reason :: atom()}.
```

Comment:
what do you think about this:
```
-spec start() -> 'ok' | {'error', 'lager_not_running'} | {'error', `no_pools_configured`}.
```
there are only two types of error so maybe we should leave it more informative? :)